README.md

```diff
@@ -161,6 +168,52 @@ Autoload can be customised using the following options:
   // routes can now be added to /defaultPrefix/something
   ```
 
+- `autoHooks` (optional) - Apply hooks from `autohooks.js` file(s) to plugins found in folder
```

Comment:
The above tree shows `autohooks.mjs` but this documentation does not mention ESM support.

Reply:
Good catch, I revised the wording and example in the autoHooks section to reflect both.